internal/util/debug/debug.go

```diff
@@ -56,11 +56,32 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 	}
 	must.NoError(statsviz.Register(http.DefaultServeMux, opts...))
 
+	http.HandleFunc("/debug/started", func(r http.ResponseWriter, _ *http.Request) {
+		// TODO https://github.com/FerretDB/FerretDB/issues/4306
+		r.WriteHeader(http.StatusOK)
+	})
+
+	// healthz handler, which is used for liveness probe, returns StatusOK when reached.
+	// This ensures that listener is running.
```

Comment:
I wouldn't say it proactively ensures that the listener is running, but when the listener is stopped, everything stops, so it's enough :) 

We can improve the comments later, for the first PR this looks enough to me.